bio/hmmer/jackhmmer/wrapper.py

```
@@ -0,0 +1,65 @@
+__license__ = "MIT"
+
+from os import path
+from snakemake.shell import shell
+
+query_file = snakemake.input.get("query")
+
+# Some parameters are mutually exclusive
+def exlusive_parameters(params):
+    num_set = 0
+    for p in params:
+        if snakemake.params.get(p) is not None:
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)